### ISSUE TITLE
atualizar tradução miui 11

### ISSUE THREAD

**Issue by maaxpane** [ingbrzy/MA-XML-10-BRAZILIAN#40]
se possivel atualizar a tradução, posso ajudar

**Comment by joaovitorpina** [comment]
Sim, também posso ajudar, tentei procurar ela nas configurações e por algum motivo ela desapareceu, só tem português de Portugal agora
